scripts/analytics/addon_snapshot.py

```diff
@@ -84,11 +84,24 @@ def get_events(self, date=None):
             connected_count = 0
             deleted_count = 0
             disconnected_count = 0
+            connected_public_count = 0
+            connected_private_count = 0
+            connected_public_affiliated_count = 0
+            connected_private_affiliated_count = 0
             node_settings_model = addon.models.get('nodesettings')
             if node_settings_model:
                 for node_settings in paginated(node_settings_model):
-                    if node_settings.owner and not node_settings.owner.all_tags.filter(name='old_node_collection', system=True).exists():
+                    node = node_settings.owner
+                    if node and not node_settings.owner.all_tags.filter(name='old_node_collection', system=True).exists():
```

Comment:
could be `if node and not node.all_tags . . .`, right?

Reply:
Yup, I forgot to update that after creating the `node` variable.